os/os.go

```diff
@@ -170,3 +170,53 @@ func (i *InstallOperation) Execute(ctx context.Context, c *internal.Clients) (*o
 	}
 	return installResp, nil
 }
+
+// ActivateOperation represents the parameters of a Activate operation.
```

Comment:
Can you move this above InstallOperation so we keep the operations alphabetized?